packages/bpk-component-nudger/readme.md

```diff
@@ -61,3 +61,6 @@ class App extends Component {
 | value                 | number                        | true     | -             |
 | className             | string                        | false    | null          |
 
+## Theme Props
+
+Same as [secondary button](/components/web/buttons#theme-props)
```

Comment:
Figured it made sense to just put a link here and keep a single source of truth.